TypeCobol.Test/Parser/Scanner/ResultFiles/PseudoText.txt

```diff
@@ -22,7 +22,6 @@
 [10,11:= ]<EqualOperator>
 [12,13:==]<PseudoTextDelimiter>
 [14,15:= ]<EqualOperator>
-Line 0[12,13] <15, Error, Tokens> - Invalid character found after pseudo-text delimiter == : expecting space, comma, semicolon or period
```

Comment:
This is not an error anymore ? It doesn't seems related to the issue.

Reply:
Hum... this is what I missed to reinterpret the case, IBM Cobol Language Reference v5.1 page 542, I think you are right, but this diagnostic was generated in case of empty placeholder. I must figure out now how to regenerate it, ok...

Reply:
done by 790bc13